hazelcast-sql-core/src/main/java/com/hazelcast/sql/impl/calcite/parse/QueryParser.java

```diff
@@ -108,4 +108,11 @@ private static Config createConfig(SqlParserImplFactory parserImplFactory) {
         }
         return configBuilder.build();
     }
+
+    private static String getTrimmedMessage(Exception e) {
```

Comment:
I find this approach dangerous because it trims all errors, respectively of their type. Some errors produced by the parser might have a line separator, which is a normal situation. Originally, we were not comfortable only with a very specific exception that produces a very long error - `org.apache.calcite.sql.parser.impl.ParseException`. We may catch exactly this exception, and trim it.